02_promise.js

```diff
@@ -0,0 +1,56 @@
+const request = require('request-promise');
+
+console.log('start ...');
+
+/**
+ * Get repos with their contributors 
+ * @param {string} organization 
+ * @param {int} limit 
+ */
+function getContributors(organization, limit = 3) {
+    return new Promise(function(resolve) {
+        function getContributorsOfReposByOwner(owner, repo) {
+            return request({
+                    "method":"GET", 
+                    "uri": `https://api.github.com/repos/${owner}/${repo}/contributors`,
+                    "json": true,
+                    "headers": {
+                        "User-Agent": "request-promise"
+                    }
+                }).then((result) => [repo, result]);
+        }
+    
+        let contributors = {};
+        request({
+            "method":"GET", 
+            "uri": `https://api.github.com/orgs/${organization}/repos`,
+            "json": true,
+            "headers": {
+                "User-Agent": "request-promise"
+            }
+        }).then((repos) => {
+            Promise.all(repos.slice(0, limit).map((repo) =>  {
```

Comment:
You need to return Promise.all and then you can use `.then((allresult)` on the same level as `.then((repos)`